Bottom: 09bcadd49897c978d21091ec740fc61bb7f0e9d2
Top:    82ee6da793ea9ebb9cb16308460c508e1fbdb574
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 12:38:10 -0700

Refresh of remove-a-smattering-of

---
diff --git a/lldb/source/Core/IOHandler.cpp b/lldb/source/Core/IOHandler.cpp
index ac7afaab0fe..5b9c760ad21 100644
--- a/lldb/source/Core/IOHandler.cpp
+++ b/lldb/source/Core/IOHandler.cpp
@@ -490,9 +490,9 @@ bool IOHandlerEditline::GetLines(StringList &lines, bool &interrupted) {
       std::string line;
       if (m_base_line_number > 0 && GetIsInteractive()) {
         if (m_output_sp) {
-          m_output_sp->GetFile().Printf("%u%s",
-            m_base_line_number + (uint32_t)lines.GetSize(),
-            GetPrompt() == nullptr ? " " : "");
+          m_output_sp->GetFile().Printf(
+              "%u%s", m_base_line_number + (uint32_t)lines.GetSize(),
+              GetPrompt() == nullptr ? " " : "");
         }
       }
